sys/kern/src/arch/arm_m.rs

```diff
@@ -1108,6 +1119,15 @@ pub unsafe extern "C" fn DefaultHandler() {
         ipsr & 0x1FF
     };
 
+    let current = CURRENT_TASK_PTR.load(Ordering::Relaxed);
+    uassert!(!current.is_null()); // irq before kernel started?
+
+    let current_prio = {
```

Comment:
**question:** The current_prio here seems like it's not being used at all? Is it intentional?